test/testenv/mcutil.go

```diff
@@ -177,3 +177,33 @@ func CheckStandalonePodOnMC(ns string, podName string) bool {
 	}
 	return found
 }
+
+// CheckMonitoringConsoleConfigured checks if mc is configured on given pod
+func CheckMonitoringConsoleConfigured(ns string, deployment *Deployment, podName string) bool {
+	output, err := exec.Command("kubectl", "logs", podName, "-n", ns).Output()
+	licenseDownloaded := false
+	nodeLicenced := false
+
+	if err != nil {
+		cmd := fmt.Sprintf("kubectl logs %s -n %s", podName, ns)
+		logf.Log.Error(err, "Failed to execute command", "command", cmd)
+		return false
+	}
+
+	for _, line := range strings.Split(string(output), "\n") {
+		// Check for empty lines to prevent an error in logic below
+		if len(line) == 0 {
+			continue
+		}
+		// Check for specifc logs that indicate that MC was configured with LM
```

Comment:
Isn't there a way to check the MC pod/container for information rather than relying on logs?

Reply:
There are two other options the way i see.....

Option1: 
Check for peers in 'splunk_monitoring_console_assets.conf' and see if License Master is one of them.

Option2:
Check 'apps/splunk_monitoring_console/local/app.conf' and see if  'is_configured = 1' is set. We can do these.

Ideally we can do all 3. Let me know

Reply:
I would go with **option 1 + option 2** & avoid the logging check to be safe. Logging with ansible can be changed anytime and configuration checks on the PVC volumes would be a good verification mechanism for the same.

Reply:
Added Option 1 + 2. And removed the log assertion